docs/proof-trace.md

```diff
@@ -25,6 +25,9 @@ Here is a BNF styled description of the format
 delimited_serial_kore := 0xffffffffffffffff serialized_term 0xcccccccccccccccc
 
 null_terminated_name := <c-style null terminated string>
+relative_position := [0-9]+(:[0-9]+)* 0x00
```

Comment:
It's not clear how this is actually formatted. Is it just a string? We can maybe make that clearer.

Reply:
My understanding from reading the code and looking at the generated files, it's just a null-terminated string that's made up of a series of non-negative integers separated by `:`

Reply:
Yeah, it's a null terminated string like `0:1:1`. I'll try to make it clearer.